### PR TITLE
add fix for "toolchain '1.52.1-aarch64-apple-darwin' is not installed"

### PR DESCRIPTION
**Pull request by torquan**
I received this error when trying to run ```./scripts/cargo-install-all.sh .```.

Solution found here: https://github.com/rust-lang/rustup/issues/1793

### CODE REVIEW

File: Solana_And_Web3/en/Section_2/Resources/m1_setup.md [hunk 1]
```diff
@@ -94,6 +94,18 @@ openssl = { version = "0.10", features = ["vendored"] }
...
 
+<details>
+<summary>Having problems with <code>toolchain</code>?</summary>
+  
+If you receive an error that looks like this – `toolchain '1.52.1-aarch64-apple-darwin' is not installed` you can try to reinstall it:
+
+```
+  rustup toolchain uninstall stable
+  rustup toolchain install stable
+```
```
Comment (adilanchian) [inline Solana_And_Web3/en/Section_2/Resources/m1_setup.md:105]: Can you do me a favor and change this to the following - 

```bash
rustup toolchain uninstall stable
rustup toolchain install stable
```

(just adding bash and then removing the indents)

**Review by adilanchian — CHANGES_REQUESTED** [review]
Good catch and thanks for adding :). 

Left some small comments for ya!

**Comment by torquan** [comment]
done!

**Review by adilanchian — APPROVED** [review]
Very nice thank you!!

**Comment by adilanchian** [comment]
ALSO CONGRATS ON YOUR FIRST PR 

😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊😊

**Comment by adilanchian** [comment]
@all-contributors please add @torquan for doc

**Pull request merged**